ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONClient.java

```diff
@@ -40,6 +46,11 @@
  */
 public class JSONClient implements IClientAPI {
 
+    private static final Logger logger = LoggerFactory.getLogger(JSONClient.class);
+
+    public final Draft draftOcppOnly =
+            new Draft_6455(Collections.emptyList(),
```

Comment:
Should be initiated in composite root (read the constructor).

Reply:
Initialization moved to constructor.